modules/apps/commerce/commerce-service/service.xml

```diff
@@ -208,6 +208,7 @@
 		<column name="paymentStatus" type="int" />
 		<column name="printedNote" type="String" />
 		<column name="purchaseOrderNumber" type="String" />
+		<column name="quoteRequested" type="boolean" />
```

Comment:
No. We dont' need that.
We just have to implement a method in the CommerceOrderImpl that will tell us if it is a "quote order" based on the status.